builddefs/bootloader.mk

```diff
@@ -199,6 +199,10 @@ endif
 ifeq ($(strip $(BOOTLOADER)), tinyuf2)
     OPT_DEFS += -DBOOTLOADER_TINYUF2
     BOOTLOADER_TYPE = tinyuf2
+    FIRMWARE_FORMAT = uf2
+
+    EEPROM_DRIVER ?= wear_leveling
+    WEAR_LEVELING_DRIVER ?= legacy
```

Comment:
Is it better to include EEPROM configs in mcu_selection.mk instead of bootloader.mk, since it does not relate to bootloader and depend more on the MCU used

Reply:
mcu_selection.mk should have zero knowledge of eeprom/bootloader.